{% extends 'base.html.twig' %} {% block title %}Hello ManyToManyController! {%
endblock %} {% block body %}
<style>
  .example-wrapper {
    margin: 1em auto;
    max-width: 800px;
    width: 95%;
    font: 18px / 1.5 sans-serif;
  }
  .example-wrapper code {
    background: #f5f5f5;
    padding: 2px 6px;
  }
</style>

<div class="example-wrapper">
  <h1>Hello {{ controller_name }}! âœ…</h1>

  <h1>{{ user1.name }}</h1>
  <ul>
    {%for f in followeds1 %}
    <li>{{ f.id }}</li>
    {%endfor%}
  </ul>

  <h1>{{ user2.name }}</h1>
  <ul>
    {%for f in followeds2 %}
    <li>{{ f.id }}</li>
    {%endfor%}
  </ul>

  <h1>{{ user3.name }}</h1>
  <ul>
    {%for f in followeds3 %}
    <li>{{ f.id }}</li>
    {%endfor%}
  </ul>

  <h1>{{ user4.name }}</h1>
  <ul>
    {%for f in followeds4 %}
    <li>{{ f.id }}</li>
    {%endfor%}
  </ul>

  This friendly message is coming from:
  <ul>
    <li>
      Your controller at
      <code>
        <a
          href="{{ 'G:/USERS/Public/TRAINING-MODULES/Symfony/00100_installation_020_dev/sy-dev-project-0090_020_MtoM/src/Controller/ManyToManyController.php'|file_link(0) }}"
          >src/Controller/ManyToManyController.php</a
        >
      </code>
    </li>
    <li>
      Your template at
      <code>
        <a
          href="{{ 'G:/USERS/Public/TRAINING-MODULES/Symfony/00100_installation_020_dev/sy-dev-project-0090_020_MtoM/templates/many_to_many/index.html.twig'|file_link(0) }}"
          >templates/many_to_many/index.html.twig</a
        >
      </code>
    </li>
  </ul>
</div>
{% endblock %}
